frontend/pages/projects/[projectId].js

```diff
@@ -91,7 +91,7 @@ export default function ProjectPage({ project, members, posts, comments, token,
       message={message.message}
       messageType={message.messageType}
       title={project ? project.name : "Project not found"}
-      description={project.shortdescription}
+      description={project?.shortdescription}
```

Comment:
@positiveimpact optional chaining to fix `ReferenceError`s is great, but WDYT about preferring throwing an error, or updating the UI when values are null / undefined instead?